### PR TITLE
Add release notes for 2018-04-05 release (3.3.33-2 and 3.4.9-2)

### PR DESCRIPTION
**Pull request by timtheisen**

### CODE REVIEW

File: docs/release/3.3/release-3-3-33-2.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+These [JIRA tickets](https://jira.opensciencegrid.org/issues/?jql=project%20%3D%20SOFTWARE%20AND%20fixVersion%20%3D%203.3.33-2%20ORDER%20BY%20priority%20DESC%2C%20key%20DESC) were addressed in this release.
+
+Detailed changes are below. All of the documentation can be found [here](../../).
+
+Updating to the new release
+---------------------------
+
+### Update Repositories
+
+To update to this series, you need [install the current OSG repositories](../../common/yum#install-osg-repositories).
```
Comment (brianhlin) [inline docs/release/3.3/release-3-3-33-2.md]: "you need to install" maybe?

And also, could you make this a site relative link? e.g. `/common/yum#install-osg-repositories`

File: docs/release/3.3/release-3-3-33-2.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+Once the repositories are installed, you can update to this new release with:
+
+``` console
+root@host # yum update
+```
+
+!!! note
+    Please be aware that running `yum update` may also update other RPMs. You can exclude packages from being updated using the `--exclude=[package-name or glob]` option for the `yum` command.
+
+!!! note
```
Comment (brianhlin) [inline docs/release/3.3/release-3-3-33-2.md]: Let's make this a list in one note

File: docs/release/3.3/release-3-3-33-2.md (hunk 1 above)
Comment (timtheisen) [reply docs/release/3.3/release-3-3-33-2.md]: OK, Let's handle that with a bulk change later.

File: docs/release/3.3/release-3-3-33-2.md (hunk 2 above)
Comment (timtheisen) [reply docs/release/3.3/release-3-3-33-2.md]: Let's handle this in a bulk change as well.

**Pull request merged**